### ISSUE TITLE
Typo in "Reversing the flow of items and accessibility" section

### ISSUE THREAD

**Issue by ttbielski** [GoogleChrome/web.dev#10102]
The second paragraph of the "Reversing the flow of items and accessibility" section has a typo, there should be "reversed row layout" instead of "reversed column layout".

**Comment by dstmarthe** [comment]
Hi, thanks for raising this. I went ahead and made this fix in staging. It won't be reflected on the site just yet as we are migrating to a new platform, but after the move it will be there. Again, thanks for pointing this out.
